lib/routes/home/floating_actions_bar.dart

```diff
@@ -140,8 +156,8 @@ class FloatingActionsBar extends StatelessWidget {
                               if (await _isBTCAddress(scannedString)) {
```

Comment:
Since this feature only applicable to btc address, I think the change should be here.
We can rename the `_isBTCAddress` to `parseBTCAddress` that would return an object with the address and amount parameter if exists. Also the parsing should be there instead in the QRCode scanner.
Parsing should use URL class and not regex, see `extractBolt11FromBip21` for example.

Reply:
[Done](https://github.com/breez/breezmobile/pull/348/commits/94c5a8c818751cbba4cba568668a20eae9bf7489)